### ISSUE TITLE
Withdrawn applications show as "Submitted on: draft"

### ISSUE THREAD

**Issue by crablab** [jac-uk/admin#637]
When an application is withdrawn, it should still show as a submitted application, but one has been withdrawn. 

Instead, it shows as a draft application. 

We should: 
- Ensure withdrawn applications are shown as submitted (and thus cannot be marked as applied)
- Clearly show on the application that it has been withdrawn. 

Eg [from here](https://admin.judicialappointments.digital/exercises/2k6Qo2JaesamXjhreHiq/applications/withdrawn/application/Pe3bIGgCx7Yi9wlJ50PF#full)

![Screenshot from 2020-07-01 12-19-16](https://user-images.githubusercontent.com/5859718/86238264-1bd9c800-bb95-11ea-9d6e-f44e519780de.png)

**Comment by crablab** [comment]
Discussion here: https://judicial-appointments.slack.com/archives/CL5U0NAF2/p1596007714022500

tl;dr: currently the application record is withdrawn, application is not updated to have status withdrawn.  This should be handled by a cloud function on `applicationRecord` update.

**Comment by warrensearle** [comment]
Hey @crablab potentially we will move this to background function in due course but this is an action within the Admin app so am happy for this to be handled on the client side for now.

i.e. when an `applicationRecord` is marked as withdrawn also mark the corresponding `application` as withdrawn

You can do this in a [batch](https://firebase.google.com/docs/firestore/manage-data/transactions#batched-writes).

**Comment by crablab** [comment]
@warrensearle I think I chatted with @YaaL about this, and the conclusion we came to is that given this is a minor change in the cloud function, we should probably prioritise that rather than "hacking" around that. I think this may come back to bite us anyway! 

What do you think? :)

**Comment by warrensearle** [comment]
I'm still happy for this to be handled within the admin app as it's a quick change, will get the bug fixed and is in line with other places in Admin and Apply where we follow a similar "double entry" pattern.

In the future we will be reviewing the patterns we use - probably we will move to doing more in cloud functions however we may decide a different approach.

**Comment by crablab** [comment]
@warrensearle Okay, sounds good :+1: 

I will fix and PR this after lunch for you :blush:

**Comment by YaaL** [comment]
@warrensearle doing this in the app will be messy, I don't see it as a quick change.
This should be done in a cloud function.
